tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -218,13 +235,23 @@ def check_meshcop_service_by_data(self, br, service_data):
         self.assertEqual(service_data['txt']['rv'], '1')
         self.assertIn(service_data['txt']['tv'], ['1.1.0', '1.1.1', '1.2.0', '1.3.0'])
 
-    def discover_all_meshcop_services(self, host):
-        instance_names = host.browse_mdns_services('_meshcop._udp')
+    def discover_all_meshcop_services(self, host, type):
```

Comment:
better to keep the original method, and add a new method:
```python
def discover_all_services(self, host, type)
```

Both `discover_all_meshcop_services` and `check_meshcop_e_service` call `discover_all_services`

Reply:
Modified accordingly, reopen if there is any improper, thanks~